File: matiasmasso/Mat.NET/DT2/DTOCobramentPerTransferencia.vb
Public Class DTOCobramentPerTransferencia

    Property User As DTOUser
    Property Delivery As DTODelivery
    Property Fch As Date
    Property Contact As DTOContact
    Property Banc As DTOBanc
    Property Concepte As String
    Property Amt As DTOAmt
    Property DocFile As DTODocFile

    Shared Function Factory(user As DTOUser,
                            delivery As DTODelivery,
                            fch As Date,
                            contact As DTOContact,
                            banc As DTOBanc,
                            concepte As String,
                            amt As DTOAmt,
                            docFile As DTODocFile) As DTOCobramentPerTransferencia

        Dim retval As New DTOCobramentPerTransferencia
        With retval
            .User = user
            .Delivery = delivery
            .Fch = fch
            .Contact = contact
            .Banc = banc
            .Concepte = concepte
            .Amt = amt
            .DocFile = docFile
        End With
        Return retval
    End Function

End Class

File: matiasmasso/Mat.NET/DTO/DTOCobramentPerTransferencia.cs
using System;

namespace DTO
{
    public class DTOCobramentPerTransferencia
    {
        public DTOUser User { get; set; }
        public DTODelivery Delivery { get; set; }
        public DateTime Fch { get; set; }
        public DTOContact Contact { get; set; }
        public DTOBanc Banc { get; set; }
        public string Concepte { get; set; }
        public DTOAmt Amt { get; set; }
        public DTODocFile DocFile { get; set; }

        public static DTOCobramentPerTransferencia Factory(DTOUser user, DTODelivery delivery, DateTime fch, DTOContact contact, DTOBanc banc, string concepte, DTOAmt amt, DTODocFile docFile)
        {
            DTOCobramentPerTransferencia retval = new DTOCobramentPerTransferencia();
            {
                var withBlock = retval;
                withBlock.User = user;
                withBlock.Delivery = delivery;
                withBlock.Fch = fch;
                withBlock.Contact = contact;
                withBlock.Banc = banc;
                withBlock.Concepte = concepte;
                withBlock.Amt = amt;
                withBlock.DocFile = docFile;
            }
            return retval;
        }
    }
}
